src/odin/server/status/jobs_info.rs

```diff
@@ -0,0 +1,22 @@
+use crate::utils::environment::fetch_var;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct JobInfo {
+  pub name: String,
+  pub enabled: bool,
+  pub schedule: String,
+}
+
+impl From<&str> for JobInfo {
+  fn from(job_name: &str) -> Self {
+    let sanitized_name = job_name.to_uppercase();
+    let enabled: bool = fetch_var(&sanitized_name, "0").eq_ignore_ascii_case("1");
+    let schedule = fetch_var(&format!("{}_SCHEDULE", &sanitized_name), "never").replace('"', "");
+    JobInfo {
+      name: job_name.to_string(),
+      enabled,
+      schedule,
+    }
+  }
+}
```

Comment:
Use `impl FromStr for ...` instead of `From<&str>`. It provides the same things as `From<&str>` along with some bonus goodies.

Reply:
Added FromStr